testing/regress/ecl/cppbody7.ecl

```diff
@@ -19,21 +19,20 @@
 
 integer4 mkRandom1 :=
 BEGINC++
+//default as 'action'
 return rtlRandom();
 ENDC++;
 
 integer4 mkRandom2 :=
 BEGINC++
-#option pure
-return rtlRandom();
+return rtlRandom(); #option pure
```

Comment:
This should give an error - as in C++ and ECL, the # must be at the start of the line (only preceding whitespace allowed)